docs/book/book.toml

```diff
@@ -8,6 +8,8 @@ title = "The Kubebuilder Book"
 google-analytics = "UA-119864590-1"
 curly-quotes = true
 additional-css = ["theme/css/markers.css", "theme/css/custom.css"]
+git-repository-url = "https://github.com/kubernetes-sigs/kubebuilder"
+edit-url-template = "https://github.com/kubernetes-sigs/kubebuilder/edit/master/docs/book/{path}"
```

Comment:
Why do we need to change the .book.tool to add a new doc section?
Please, check that we just add the file and we create it into the Summary.

Please, look at how the other docs/sections are done.

Reply:
To add git-repo and git-edit icons on the top-right page, to let other users to go to KB project quickly, and go to the edit page easily if want to modify the doc.